l1-contracts/deploy-scripts/DeployUtils.s.sol

```diff
@@ -445,13 +445,13 @@ abstract contract DeployUtils is Script {
                     config.contracts.governanceMinDelay
                 );
         } else if (compareStrings(contractName, "ChainAdminOwnable")) {
-            return abi.encode(config.ownerAddress, address(0));
+            return abi.encode(config.ownerAddress, address(0), address(0));
```

Comment:
It should not be 0 here, or in other places. The chainAdmin is deployed after the CTM ( almost certainly) so you should be able to take it out of the config.

Reply:
Its zero everywhere, that is not good